test/e2e_node/local_storage_allocatable_eviction_test.go

```diff
@@ -94,7 +94,7 @@ var _ = framework.KubeDescribe("LocalStorageAllocatableEviction [Slow] [Serial]
 		// Set up --kube-reserved for scratch storage
 		tempSetCurrentKubeletConfig(f, func(initialConfig *kubeletconfig.KubeletConfiguration) {
 			framework.Logf("Set up --kube-reserved for local storage reserved %dMi", diskReserve)
-			initialConfig.KubeReserved = kubeletconfig.ConfigurationMap(map[string]string{"storage": fmt.Sprintf("%dMi", diskReserve)})
```

Comment:
Should we get this merged in 1.8 ? Since we have changed the resource name check case
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubelet/app/server.go#L774

Reply:
yes, I think so

Reply:
This is fixed in [Refactor eviction tests](https://github.com/kubernetes/kubernetes/pull/52373)